src/azure-cli/azure/cli/command_modules/resource/_help.py

```diff
@@ -788,6 +788,76 @@
 short-summary: Show a deployment operation at tenant scope.
 """
 
+helps['deployment-scripts'] = """
+type: group
+short-summary: Manage deployment scripts at subscription or resource group scope.
+"""
+
+helps['deployment-scripts list'] = """
+type: command
+short-summary: List all deployment scripts.
+parameters:
+  - name: --subscription
```

Comment:
`--subscription` is a global parameter which is already defined globally. Do you want to overwrite it here?
 
![image](https://user-images.githubusercontent.com/49508232/79421867-4ed8da80-7fee-11ea-91aa-aeffc1c715fb.png)